test-jdk8/pom.xml

```diff
@@ -0,0 +1,40 @@
+<?xml version="1.0" encoding="UTF-8"?>
```

Comment:
wondering why `jdk8` and not `11` and why is it not included in the `modules`

Reply:
Because these tests run *only* on JDK 8. They are not compilable/executable with JDK11

Module is included, but in a profile:

```
    <profile>
      <id>jdk8</id>
      <modules>
        <module>test-jdk8</module>
      </modules>
      <activation>
        <jdk>1.8</jdk>
      </activation>
    </profile>
```

See https://speakerdeck.com/onenashev/leap-from-java-8-to-java-11-in-jenkins-our-war-story?slide=53 and https://speakerdeck.com/onenashev/leap-from-java-8-to-java-11-in-jenkins-our-war-story?slide=54 for the approach explanation